README.md

```diff
@@ -15,9 +15,13 @@ npm install --save redux-ignore
 ## API
 
 ```js
-import ignoreActions from 'redux-ignore';
+import ignoreActions, {filterActions} from 'redux-ignore';
```

Comment:
Maybe we should still encourage using `import { ignoreActions, filterActions }`? It looks cleaner when using both.